frontend/src/DiagramList/DiagramList.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { Link } from "react-router-dom";
+
+import "./DiagramList.scss";
+
+export default function DiagramList({ diagrams, joinDiagram }) {
+  function displayDiagramList() {
+    if (!diagrams) {
+      return <p>Loading diagrams...</p>;
+    }
+    if (diagrams.length === 0) {
+      return <p>There are no diagrams. Create one now!</p>;
+    }
+
+    return diagrams.map((diagramId) => (
+      <Link
+        to={`/diagrams/${diagramId}`}
+        key={diagramId}
+        className="diagram-item"
+      >
+        <h3 className="title">{diagramId}</h3>
+        <p className="created">Created on: 01 Feb 2020</p>
```

Comment:
do diagrams not a date created attribute?

Reply:
maybe we also want, last edited date?

Reply:
they don't have one for now, I'll add one as part of a separate task